src/test/java/api/loans/ClaimItemReturnedAPITests.java

```diff
@@ -9,7 +9,7 @@
 import static org.folio.circulation.domain.representations.LoanProperties.ACTION;
 import static org.folio.circulation.domain.representations.LoanProperties.ACTION_COMMENT;
 import static org.folio.circulation.domain.representations.LoanProperties.CLAIMED_RETURNED_DATE;
-import static org.folio.circulation.resources.ClaimItemReturnedResource.ITEM_CLAIMED_RETURNED_DATE;
+import static org.folio.circulation.domain.ClaimItemReturnedRequest.ITEM_CLAIMED_RETURNED_DATE;
```

Comment:
I don't think the API tests should rely on the internal details of the domain, as it hides the impact if they change or are incorrect.